isample/vocabulary/sampledfeature/1.0/.htaccess

```diff
@@ -2,23 +2,23 @@ Options +FollowSymLinks
 RewriteEngine on 
 
 #  uri to access individual term
-# https://vocabs.ardc.edu.au/repository/api/lda/isamples/isamples-sampled-feature-type-vocabulary/1-0/resource?uri=https://w3id.org/isample/vocabulary/sampledfeature/1.0/atmosphere
 
 RewriteCond %{HTTP:Accept} text/turtle [NC]
-RewriteRule       ^(.+)$     https://vocabs.ardc.edu.au/repository/api/lda/isamples/isamples-sampled-feature-type-vocabulary/1-0/resource.ttl?uri=https://w3id.org/isample/vocabulary/sampledfeature/1.0/$1 [R=303,L]
+RewriteRule       ^(.+)$   https://vocabs.ardc.edu.au/repository/api/lda/isamples/isamples-sampled-feature-type-vocabulary/1-0/resource.ttl?uri=https://w3id.org/isample/vocabulary/sampledfeature/1.0/$1 [R=303,L]
 
 RewriteCond %{HTTP:Accept} application/json [NC]
-RewriteRule       ^(.+)$               https://vocabs.ardc.edu.au/repository/api/lda/isamples/isamples-sampled-feature-type-vocabulary/1-0/resource.json?uri=https://w3id.org/isample/vocabulary/sampledfeature/1.0/$1 [R=303,L]
+RewriteRule       ^(.+)$   https://vocabs.ardc.edu.au/repository/api/lda/isamples/isamples-sampled-feature-type-vocabulary/1-0/resource.json?uri=https://w3id.org/isample/vocabulary/sampledfeature/1.0/$1 [R=303,L]
 
 RewriteCond %{HTTP:Accept} application/rdf+xml [NC]
-RewriteRule       ^(.+)$               https://vocabs.ardc.edu.au/repository/api/lda/isamples/isamples-sampled-feature-type-vocabulary/1-0/resource.rdf?uri=https://w3id.org/isample/vocabulary/sampledfeature/1.0/$1 [R=303,L]
+RewriteRule       ^(.+)$   https://vocabs.ardc.edu.au/repository/api/lda/isamples/isamples-sampled-feature-type-vocabulary/1-0/resource.rdf?uri=https://w3id.org/isample/vocabulary/sampledfeature/1.0/$1 [R=303,L]
 
 RewriteCond %{HTTP:Accept} text/html [NC]
-RewriteRule       ^(.+)$               https://vocabs.ardc.edu.au/repository/api/lda/isamples/isamples-sampled-feature-type-vocabulary/1-0/resource.html?uri=https://w3id.org/isample/vocabulary/sampledfeature/1.0/$1 [R=303,L]
+RewriteRule       ^(.+)$   https://vocabs.ardc.edu.au/repository/api/lda/isamples/isamples-sampled-feature-type-vocabulary/1-0/resource.html?uri=https://w3id.org/isample/vocabulary/sampledfeature/1.0/$1 [R=303,L]
 
 # default format is html
-RewriteRule       ^(.+)$               https://vocabs.ardc.edu.au/repository/api/lda/isamples/isamples-sampled-feature-type-vocabulary/1-0/resource?uri=https://w3id.org/isample/vocabulary/sampledfeature/1.0/$1 [R=303,L]
+
+RewriteRule       ^(.+)$   https://vocabs.ardc.edu.au/repository/api/lda/isamples/isamples-sampled-feature-type-vocabulary/1-0/resource?uri=https://w3id.org/isample/vocabulary/sampledfeature/1.0/$1 [R=303,L]
 
 # no term, just the vocab
-RewriteRule       ^$
-https://vocabs.ardc.edu.au/repository/api/lda/isamples/isamples-sampled-feature-type-vocabulary/1-0/resource?uri=https://w3id.org/isample/vocabulary/sampledfeature/1.0/sampledfeaturevocabulary  [R=303,L]
```

Comment:
This is failing because the rule is split on two lines.  You need a trailing `\` to continue to next line.  It's a syntax error as is and maybe causing the whole file to fail.  Error messages are like this if you try locally:
```
[Tue Jan 02 16:47:27.440988 2024] [core:alert] [pid XXX] [client XXX] /XXX/w3id.org/isample/vocabulary/sampledfeature/1.0/.htaccess: RewriteRule: bad argument line '^$'
```